docker/back/api/nginx/vhost.conf

```
@@ -0,0 +1,25 @@
+server_tokens off;
+
+log_format docker '$remote_addr - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_time "$http_x_forwarded_for"';
```

```suggestion
log_format docker '$remote_addr - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_time "$http_x_forwarded_for" $request_uid';
```

Ajout de l'ID de requête dans les logs

---

done